tests/unit_tests/llms/test_lllm_class.py

```diff
@@ -0,0 +1,55 @@
+"""Fake LLM wrapper for testing purposes."""
+from typing import List, Optional
+
+import pytest
+
+from langchain.callbacks.manager import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
+from langchain.llms.base import LLM
+from langchain.schema import LLMResult
+
+
+class FakeNewSignLLM(LLM):
```

Comment:
the whole point of a FakeLLM is to use in other tests right? at very least, if we think this is useful, we should put in the fake_llm file with the other fake llm